cmd/uhc/get/cmd.go

```diff
@@ -72,11 +72,36 @@ func init() {
 
 func run(cmd *cobra.Command, argv []string) {
 	// Check that there is exactly one command line parameter:
-	if len(argv) != 1 {
-		fmt.Fprintf(os.Stderr, "Expected exactly one argument\n")
+	if len(argv) < 1 || len(argv) > 2 {
+		cmd.Usage()
+		fmt.Fprintf(os.Stderr, cmd.Usage().Error())
 		os.Exit(1)
 	}
-	path := argv[0]
+	preParsePath := argv[0]
+	path := preParsePath
+
+	switch preParsePath{
+
+	// list resources
+
+	case "accounts", "accts":
+		path = "/api/accounts_mgmt/v1/accounts"
+	case "subscriptions", "subs":
+		path = "/api/accounts_mgmt/v1/subscriptions"
+	case "organizations", "orgs":
+		path = "/api/accounts_mgmt/v1/organizations"
+
+	// get individual resources
+
+	case "account", "acct":
+		path = "/api/accounts_mgmt/v1/accounts/" + argv[1]
```

Comment:
Here you are accessing index 1, but you didn't check that it actually exists. That will cause a panic if the user types `uhc get account`, without the account identifier, for example. Can you add that check?

Reply:
validation added